Source/Krypton Components/Krypton.Toolkit/General/Toast Notification/KryptonBasicToastNotificationData.cs

```diff
@@ -0,0 +1,30 @@
+﻿#region BSD License
+/*
+ *
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
+ *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2024 - 2024. All rights reserved.
+ *
+ */
+#endregion
+
+namespace Krypton.Toolkit
+{
+    public struct KryptonBasicToastNotificationData
+    {
+        #region Public
+
+        public bool? ShowCloseBox { get; set; }
+
+        public int? CountDownSeconds { get; set; }
+
+        public string NotificationContent { get; set; }
```

Comment:
This should be nullable as well, As there might be title only

Reply:
Now nullable